### PR TITLE
Update bio

### PR DESCRIPTION
**Pull request by ana0**

### CODE REVIEW

File: about.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ Our Networks organizers have hosted civic tech events and community networks wor
...
 </ul>
 
-**Sarah Friend**{:#sarah} is an artist and software engineer working at a large blockchain development studio. When not doing that, she creates games and other interactive experiences. Her practice more broadly explores the polar concerns of privacy and transparency and the political/environmental implications of technology. She is a proud Recurse Center alum, and has recently presented at the Montreal International Games Showcase, the MoneyLab program by the Institute for Networked Culture in London, and Transmediale Berlin.
+**Sarah Friend**{:#sarah} is an artist and software engineer, with special interest in blockchain and the p2p web. When not doing that, she creates games and other interactive experiences. Her practice investigates murky dichotomies - like those between privacy and transparency, centralization and decentralization, and the environment and technology - with playfulness and absurdist humour. She is a proud Recurse Centre alum, and has recently exhibited work at NEoN Festival in Scotland, Moneylab in London, Gray Area Festival in San Francisco, Microwave Festival in Hong Kong, the Athens Biennale, and the ZKM Center for Art and Media In Germany.
```
Comment (dcwalk) [inline about.md]: Looks great!

Typographical q: do we want to change the hyphen "-" to an emdash? cc @garrying

Comment (dcwalk) [reply about.md]:
```suggestion
**Sarah Friend**{:#sarah} is an artist and software engineer, with special interest in blockchain and the p2p web. When not doing that, she creates games and other interactive experiences. Her practice investigates murky dichotomies &mdash; like those between privacy and transparency, centralization and decentralization, and the environment and technology &mdash; with playfulness and absurdist humour. She is a proud Recurse Centre alum, and has recently exhibited work at NEoN Festival in Scotland, Moneylab in London, Gray Area Festival in San Francisco, Microwave Festival in Hong Kong, the Athens Biennale, and the ZKM Center for Art and Media In Germany.  
```

Comment (ana0) [reply about.md]: Changing seems good to me :)

File: about.md [hunk 2]
```diff
@@ -43,7 +43,7 @@ Our Networks organizers have hosted civic tech events and community networks wor
...
 
-**Sarah Friend**{:#sarah} is an artist and software engineer working at a large blockchain development studio. When not doing that, she creates games and other interactive experiences. Her practice more broadly explores the polar concerns of privacy and transparency and the political/environmental implications of technology. She is a proud Recurse Center alum, and has recently presented at the Montreal International Games Showcase, the MoneyLab program by the Institute for Networked Culture in London, and Transmediale Berlin.
+**Sarah Friend**{:#sarah} is an artist and software engineer, with special interest in blockchain and the p2p web. When not doing that, she creates games and other interactive experiences. Her practice investigates murky dichotomies &mdash; like those between privacy and transparency, centralization and decentralization, and the environment and technology &mdash; with playfulness and absurdist humour. She is a proud Recurse Centre alum, and has recently exhibited work at NEoN Festival in Scotland, Moneylab in London, Gray Area Festival in San Francisco, Microwave Festival in Hong Kong, the Athens Biennale, and the ZKM Center for Art and Media In Germany.
```
Comment (garrying) [inline about.md]:
```suggestion
**Sarah Friend**{:#sarah} is an artist and software engineer, with special interest in blockchain and the p2p web. When not doing that, she creates games and other interactive experiences. Her practice investigates murky dichotomies&mdash;like those between privacy and transparency, centralization and decentralization, and the environment and technology&mdash;with playfulness and absurdist humour. She is a proud Recurse Centre alum, and has recently exhibited work at NEoN Festival in Scotland, Moneylab in London, Gray Area Festival in San Francisco, Microwave Festival in Hong Kong, the Athens Biennale, and the ZKM Center for Art and Media In Germany.  
```

Comment (garrying) [reply about.md]: Removing white-space around the em dashes

Comment (dcwalk) [reply about.md]: Right, that is practice around emdashes! Gonna do a quick commit then ping you folks back,

**Comment by dcwalk** [comment]
@garrying and @ana0 good for final merge?

**Review by garrying — APPROVED** [review]
Minor thought: `Recurse Centre` -> `Recurse Center` ?

**Comment by ana0** [comment]
@garrying Good catch, updated

**Comment by dcwalk** [comment]
This looks good and is approved/working so let's merge!

**Pull request merged**